test/e2e/move.update.police.test.js

```diff
@@ -92,4 +93,12 @@ if (FEATURE_FLAGS.EDITABILITY) {
       await checkUpdateMoveDetails()
     }
   )
+
+  test('User should be able to change move date to tomorrow', async () => {
+    await checkUpdateMoveDate()
+  })
+
+  test('User should be able to change move date to another date', async () => {
+    await checkUpdateMoveDate(3)
```

Comment:
This felt a little hard to read just looking at this call. It wasn't easy to tell `3` was going to set a custom date, 3 days from now.

I wonder if it might be a bit simpler to just use a faker date in the future in the `fillInDate` method and not worry about needing to define how many days away it is being changed to.

Reply:
Personally, I'd like to use something like 'moment` and be able to pass stuff like `+3d`. (But didn't want to introduce or reinvent it here). Will move the number stuff out.